tests/test_analysis/test_feature_relevance/test_relevance_table.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.tree import DecisionTreeRegressor
+
+from etna.analysis.feature_relevance import get_model_relevance_table
+from etna.analysis.feature_relevance import get_statistics_relevance_table
+
+
+@pytest.mark.parametrize(
+    "method,method_kwargs",
+    ((get_statistics_relevance_table, {}), (get_model_relevance_table, {"model": DecisionTreeRegressor()})),
+)
+def test_interface(method, method_kwargs, simple_df_relevance):
+    relevance_table = method(simple_df_relevance[0], simple_df_relevance[1], **method_kwargs)
```

Comment:
May be it will be simpler to read if `simple_df_relevance` will be unpacked into `df` and `df_exog`. I also suggest to use here `df=...` and `df_exog=...` instead of positional parameters.